.github/scripts/build_daprbundle.py

```diff
@@ -120,12 +124,18 @@ def downloadBinaries(dir):
     unpack_archive(cli_filepath,dir)
     os.remove(cli_filepath)
 
+# Returns the fileName of the docker image to be saved. e.g. for image `daprio/dapr:1.7.0`, the fileName would be `dapr-1.7.0.tar.gz` i.e. without owner prefix
+def getFileName(image):
+    imageBase = image.split("/")[-1]
+    fileName = imageBase.replace(":","-") + ".tar.gz"
+    return fileName
 
+# Downloads the givern version of docker image and saves it in `out_dir` folder
 def downloadDockerImage(image_name, version, out_dir):
     docker_image=f"{image_name}:{version}"
     if (version == "latest"):
         docker_image =image_name
-    fileName= docker_image.replace("/","-").replace(":","-") + ".tar.gz"
+    fileName = getFileName(docker_image)
     downloadPath = os.path.join(out_dir,fileName)
 
     if not os.path.exists(out_dir):
```

Comment:
@shivamkm07 Once image is pulled, you have to retag it to a particular name before saving it ... 
The issue here is 
the image is pulled as `daprio/dapr:<version>.tar.gz`. 
When `docker save` is used to save an image to file ... the details about the repository, tag imageId, size, created at etc. all details are also saved as part of the tar.gz file. 

When docker load is used on the tar.gz file, the same image is loaded back with the same details. 

So even if the filename changes, the image name will still remain `daprio/dapr:<version>`.

For this one fix that I was able to do was once the image is pulled and before it is saved, 
run `docker tag daprio/dapr:<> dapr/dapr:<version>`. 

And then save that image.... `dapr/dapr:<version>`. 

Only then change we have the same name be used to again run the image. 

Consequently change the file name it is saved as `dapr-dapr-<version>.tar.gz`. As both the filename and image name needs to be one or that is what is assumed for now in CLI.